src/structures/Guild.js

```diff
@@ -573,7 +573,9 @@ class Guild {
    * be resolved, the user ID will be the result.
    * @example
    * // ban a user
-   * guild.ban('123123123123');
+   * guild.ban(user)
```

Comment:
But you don't explain what "user" is at all.
At the same time you add a promise where you pass the user if it resolves and console log it, but guild.ban can be used without the user being on the server initially, so you would have no user to resolve.